pollination_handlers/inputs/postprocess.py

```diff
@@ -0,0 +1,53 @@
+"""Handlers for post-processing options."""
+import json
+from jsonschema import validate
+
+from .helper import get_tempfile
+
+
+def grid_metrics(json_file):
+    """Validate the file for custom grid metrics.
+
+        Args:
+            json_file: A JSON file with custom grid metrics.
+
+        Returns:
+            str -- Path to a the custom grid metrics file.
+    """
+    _of_schema = {
+        'type': 'array',
+        'items': {
+            'properties': {
+                'minimum': {'type': 'number'},
+                'maximum': {'type': 'number'},
+                'exclusiveMinimum': {'type': 'number'},
+                'exclusiveMaximum': {'type': 'number'}
+            },
+            'additionalProperties': False
+        }
+    }
+    schema = {
+        'type': 'array',
+        'items': {
+            'type': 'object',
+            'properties': {
+                'minimum': {'type': 'number'},
+                'maximum': {'type': 'number'},
+                'exclusiveMinimum': {'type': 'number'},
+                'exclusiveMaximum': {'type': 'number'},
+                'allOf': _of_schema,
+                'anyOf': _of_schema
+            },
+            'additionalProperties': False
+        }
+    }
+
+    with open(json_file) as file:
+        grid_metrics = json.load(file)
```

Comment:
I imagine we want to let the user input their criteria as a text in a Grasshopper panel or a text in Lavender. Editing a file for this input can be time-consuming.